docs/classic-ui/images.md

```diff
@@ -254,81 +254,199 @@ The `ImageScaling` view explicitly checks the permissions of the current user.
 To access image scales, which are normally not accessible to the current user, override the `validate_access` method in `plone.namedfile.scaling.ImageScale`.
 
 
-## `srcset` configuration
+(classic-ui-images-responsive-image-support)=
+## Responsive image support
 
-In `/@@imaging-controlpanel` Plone allows you to define HTML {term}`srcset` attributes.
-A `srcset` can help the browser serve the best fitting image for the current users situation.
-Which image scale is best for the user can be decided on different metrics.
+Plone supports the generation of picture tags with `srcset`'s for image optimization.
+Additionally you can define `media queries` for {ref}`classic-ui-images-responsive-image-support-art-direction` and further optimization.
```

Comment:
The `ref` renders as "Art direction", which is annoying. We can fall back to MyST linking.

```suggestion
Additionally you can define [media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) for [art direction](classic-ui-images-responsive-image-support-art-direction) and further optimization.
```